tests/ast/test_ast_driver.py

```diff
@@ -119,57 +119,63 @@ def run_test(session):
             [2, "two"],
             [3, "three"],
```

Comment:
Changing here emit mode to False, so the tested functions are the first time when any ast related functionality is invoked. Makes debugging easier.